new_game.py

```diff
@@ -0,0 +1,139 @@
+# Python 2
+from PyDictionary import PyDictionary
+import random, time, os
+
+class Data(object):
+
+    # Builds the word database
+    def collect_words(word_file="gre_word_file.txt"):
+        wordlist = open(word_file).read().splitlines()
+        return wordlist
+
+    def __init__(self, words=collect_words()):
+        self.words = words
+        self.lookup = PyDictionary()
+
+    def display_words(self):
+        for word in self.words:
+            print word
+            
+    # Chooses a random word from the wordlist
+    def answer(self):
+        answer_idx = random.randint(0, len(self.words) - 1)
+        answer = self.words[answer_idx]
+        del self.words[answer_idx]
+        return answer
+        
+    # Queries the definition of the answer
+    def definition(self, answer):
+        query = self.lookup.meaning(answer)
+        print '\nDefinition: \n'
+        for definition in query:
+            print definition, '\n', query[definition]
+        print '-' * 75 + '\n'
+    # Builds a list consisting of the answer and 3 other random words
+    def choices(self,answer):
+        my_choices = [answer]
+        while len(my_choices) < 4:
+            choice = random.choice(self.words)
+            if choice not in my_choices:
+                my_choices.append(choice)
+        random.shuffle(my_choices)
+        print my_choices
+    # Prompts user to type the answer 3x if the guess is incorrect
+    def practice(self, answer):
+        print 'Please type the answer 3x, each on its own line.\n'
+        count = 0
+        while count < 3:
+            word = raw_input('> ')
+            if word == answer:
+                count += 1
+            else:
+                print 'Make sure your spelling is correct.'
+        game_on = False
+
+
+class Main(object):
+
+    def __init__(self):
+        self.wins = 0
+        self.attempts = 0
+
+    def clear_screen(self):
+        if os.name == "posix":
+            # Unix/Linux/MacOS/BSD/etc
+            os.system('clear')
+        elif os.name in ("nt", "dos", "ce"):
+            # DOS/Windows
+            os.system('CLS')
+        else:
+            # Fallback for other operating systems.
+            print '\n' * numlines
+    # Prompt to choose easy = number version or hard = word version
+    def start(self):
+        self.clear_screen()
+        game_on = True
+        while game_on:
+            choice = raw_input('\nWould you like the easy or hard version? \n').lower()
+            if choice.startswith('h'):
```

Comment:
You should just expect "easy" and "hard". not just words that start with e and h